lhcb-YYYY-open-data-curation/scripts/README.md

```diff
@@ -0,0 +1,39 @@
+# How to prepare the metadata for an LHCb OD release
```

Comment:
Currently the PR has:

- lhcb-YYYY-open-data-curation/README.md
- lhcb-YYYY-open-data-curation/scripts/README.md

What about creating only one README file living in `lhcb-YYYY-open-data-curation` directory that would contain everything people need to know when working with scripts, and leave the `scripts` directory without any README?

(That's the technique we use for other data curation campaigns.)

Reply:
Ok, will fix.